codestodians/food-basket-seller
# Chore/basic configuration 153737385

#### What will this PR do?
- Add necessary configuration to the application.

#### Description of Task to be completed?
- Initialize node using `yarn` as the package manager.
- Setup `babel` for Transpiling.
- Configure eslint.
- Setup `HoundCI`
- Setup `TravisCI`
- Setup code coverage with coveralls

#### How should this be manually tested?
- Check the `checks` that must pass before approval of a PR; Hound, Travis and Coveralls should be included.
- Check out into the `chore/basic-configuration-153737385` branch and see that Transpiling and linting work.

#### Any background context you want to provide?
- With this PR comes the basic setup that will ensure strict adherence to style guides and best practices going forward.

#### What are the relevant pivotal tracker stories?
- [#153737385](https://www.pivotaltracker.com/story/show/153737385)
